terminado/management.py

```diff
@@ -206,8 +219,9 @@ def on_eof(self, ptywclients):
 
     def pty_read(self, fd, events=None):
         """Called by the event loop when there is pty data ready to read."""
-        r, _, _ = select.select([fd], [], [], .1)
-        if not r:
+        # prevent blocking on fd
+        if not _poll(fd, timeout=0.1):  # 100ms
+            self.log.info(f"Spurious pty_read() on fd {fd}")
```

Comment:
Hello! Could you please log the output via `.debug(...)` here instead of `.info(...)`?

Reply:
done.